pubspec.yaml

```diff
@@ -3,6 +3,10 @@ description: A Flutter plugin for Android and iOS supports cropping images
 version: 1.3.1
 homepage: https://github.com/hnvn/flutter_image_cropper
 
+environment:
+  sdk: '>=2.12.0-0 <3.0.0'
+  flutter: ^1.12.13
```

Comment:
Change Flutter version ?

Reply:
Which Flutter version should I change to?

Reply:
Since null-safety was introduced by 1.24.0-12.2.pre, I guess this should be the minimum Flutter requirement but not sure. Do Do you know if we can run an older Flutter SDK with a more recent Dart version ?